Dockerfile-devcore-centos-7

```diff
@@ -8,12 +8,14 @@ RUN yum install -y curl wget which
 RUN yum install -y sudo
 
 # ENVIRONMENT
-RUN yum install -y git perl ruby
+RUN yum install -y perl ruby
 RUN yum install -y perl-Data-Dumper perl-IPC-Cmd
 RUN yum install -y gcc gcc-c++ make
 RUN yum install -y java-1.8.0-openjdk ant ant-junit maven
 RUN yum install -y rpm-build createrepo rsync
 RUN curl https://rclone.org/install.sh | sudo bash
+RUN yum -y install https://packages.endpointdev.com/rhel/7/os/x86_64/endpoint-repo.x86_64.rpm
```

Comment:
`RUN yum -y install`
to
`RUN yum install -y`

Reply:
done